schema/README.md

```diff
@@ -740,7 +704,7 @@ An example of a `Copyright` and a derived `Right`:
     ],
     "@type": "<coalaip placeholder>/Right",
     "@id": "<URI pointing to this object>",
-    "source": "<URI pointing to a Copyright object>",
+    "rightsOf": "<URI pointing to a Copyright object>",
```

Comment:
For example this line

Reply:
I hope I made this more clear in 5343372

Reply:
Yes! Okay I understand now. Looks good to me!